### DISCUSSION TITLE
Feature request: Add gists to profiles

### DISCUSSION THREAD

**Discussion by lcherone** [community/community#22691] [New to GitHub]
<p>For some reason the gists feature on github feels disconnected from the main github, but if you goto your gists profile its much like the github profile</p>
<p><a href="https://gist.github.com/lcherone" rel="nofollow noopener">https://gist.github.com/lcherone</a> vs <a href="https://github.com/lcherone" rel="nofollow noopener">https://github.com/lcherone</a></p>
<p>It would be nice to see Gists as a tab in the profile section one day, perhaps next to the shiny new Projects tab.</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/lcherone">@lcherone</a>,</p>
<p>It’s not quite what you’re asking for, but you can <a href="https://github.blog/changelog/2019-03-13-pinned-gists/" rel="nofollow noopener">pin gists</a> on your profile page</p>

**Comment by namir-ahmed** [comment]
<p>I agree, this feature is absolutely nessessary. There is no reason to have gists so seperate from your main profile. There have been a few times when I wanted to put a small script on gist that others could use, but if anyone visits my profile, then they would have no way of know I made anything. Another tab for gists would be great since as of now, there is no elegant way of showcasing your gists.</p>

**Comment by MChevy422** [comment]
<p>Hello <span><a class="mention" href="/lcherone">@lcherone</a></span>, <br><br>Thank you for your feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list. Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.<br><br>Once again, thank you for your input!<br><br>Greatly appreciated<br><br>MChevy422</p>

**Comment by patricktokeeffe** [comment]
<p>I joined this community to make this precise request. Currently, there are zero ways to access a user’s gists from their profile page. Not one single link, anywhere.</p>
<p>In contrast, the user profiles displayed on <em><a href="http://gist.github.com" rel="nofollow noopener">gist.github.com</a></em> have a giant green button sending visitors back to Github. And the page header has a permalink “Return to Github”. Feels pretty one-sided</p>

**Comment by nobewayo** [comment]
4 years later and this is still not a feature? Why have it so spererated from everything else on Github?

**Comment by boexler** [comment]
I made a Userscript to add a **View Gists Profile** Button.

https://gist.github.com/djonasdev/55f5cc1fc8462ec8b3fbd0af7c5bc50a#file-github-gists-profile-button-user-js

![grafik](https://github.com/user-attachments/assets/5cd1c9d5-13eb-40c4-8390-bf4be7d4ee45)

![grafik](https://github.com/user-attachments/assets/41c31d8b-8035-4c20-8194-bf2b0f54d26b)
